.github/workflows/welcome-first-time-contrib.yml

```
@@ -17,7 +17,7 @@ jobs:
     if: ${{ !contains(fromJson('["asyncapi-bot", "dependabot[bot]", "dependabot-preview[bot]", "allcontributors[bot]"]'), github.actor) }}
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/github-script@v6
+    - uses: actions/github-script@v7
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)